File: ProAgil-App/src/app/_models/Evento.ts
import { Lote } from './Lote';
import { Palestrantes } from './Palestrantes';
import { RedeSocial } from './RedeSocial';

export interface Evento {
    id: number;

    local: string;

    dataEvento: Date;

    tema: string;

    qtdPessoas: number;

    imageURL: string;

    telefone: string;

    email: string;

    lotes: Lote[];

    redesSociais: RedeSocial[];

    palestranteEventos: Evento[];

    eventoid: number;
}

File: ProAgil.API/Models/Evento.cs
namespace ProAgil.API.Models
{
    public class Evento
    {
        public int Eventoid {get; set;}

        public string Local { get; set; }

        public string DataEvento { get; set; }

        public string Tema { get; set; }

        public int QtdPessoas { get; set; }

        public string Lote { get; set; }
    }
}